lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -14,14 +14,27 @@ class BulkDataPatientExportSequence < SequenceBase
       requires :token
       conformance_supports :Patient
 
-      def export_kick_off(klass)
-        headers = { accept: 'application/fhir+json', prefer: 'respond-async' }
+      def assert_export_kick_off(klass)
+        reply = export_kick_off(klass)
 
-        url = ''
-        url += "/#{klass}" if klass.present?
-        url += '/$export'
+        assert_response_accepted(reply)
+        @content_location = reply.response[:headers]['content-location']
 
-        @client.get(url, @client.fhir_headers(headers))
+        assert @content_location.present?, 'Export response header did not include "Content-Location"'
+      end
+
+      def assert_export_status(content_location = @content_location)
+        reply = export_status_check(content_location)
+
+        assert reply.code == 200, "Bad response code: expected 200, 202, but found #{reply.code}."
```

Comment:
Your description does not match the value checked.

Reply:
The status check response shall be either 202 (still processing) or 200 (ready). Status 202 is handled inside export_status_check function. The logic inside that function is that if the status code is 202 then wait and check status again. If the status code is not 202 then return. This assert function here only need to check 200.

Reply:
Ah, ok.  Its good that it prints out something that would be logical to the user, but on the surface it looks like an oversight in the code.  I think a comment is necessary here would save confusion later.